js/src/views/edit-attribute-field-attachment.js

```diff
@@ -103,12 +103,13 @@ var editAttributeFieldAttachment = sui.views.editAttributeField.extend( {
 
 		} else {
 
-			attachmentThumb = (typeof attachment.sizes.thumbnail !== 'undefined') ?
-				attachment.sizes.thumbnail :
-				_.first( _.sortBy( attachment.sizes, 'width' ) );
+			attachmentThumb = (
+				typeof attachment.sizes !== 'undefined' &&
+				typeof attachment.sizes.thumbnail !== 'undefined'
+			) ? attachment.sizes.thumbnail : _.first( _.sortBy( attachment.sizes, 'width' ) ) || {};
 
 			jQuery( '<img/>', {
-				src:    attachmentThumb.url,
+				src:    attachmentThumb.url || attachment.url,
```

Comment:
It looks like this is still going to be throwing a lot of warnings about trying to get non-existant properties ('url', 'width', 'height') of `attachmentThumb` when it's an empty object.